docs/general/colors.md

```diff
@@ -84,6 +84,22 @@ const options = {
 
 :::
 
+### Dynamic datasets at runtime
+
+By default the colors plugin only works when you initialize the chart without any colors for the border or background specified. 
+If you want to force the colors plugin to always color your datasets, for example when using dynamic datasets at runtime you will need to set the `forceColorsPlugin` option to true:
+
+```js
+const options = {
+  plugins: {
+    colors: {
+      forceColorsPlugin: true
```

Comment:
Maybe just `force`? "path" to the property already contains "plugins" and "colors" (`plugins.colors.force`)

Reply:
`forceOverride` would be more desciptive then I guess